xapi.spec.in

```diff
@@ -359,6 +365,11 @@ esac
 /usr/bin/xe
 /etc/bash_completion.d/xe
 
+%files gpg-keys
+%config(noreplace) @OPTDIR@/gpg/pubring.gpg
+%config(noreplace) @OPTDIR@/gpg/secring.gpg
+%config(noreplace) @OPTDIR@/gpg/trustdb.gpg
+
```

Comment:
http://www-uxsup.csx.cam.ac.uk/~jw35/docs/rpm_config.html provides a good overview what `(noreplace)` does. In short: only if the user edits these files, they are not overwritten during an upgrade.